src/main/java/domain/WinningNumbers.java

```diff
@@ -5,9 +5,6 @@
 
 public class WinningNumbers {
 
-    private static final int WINNING_TICKET_SIZE = 6;
-
-    private static final String WINNING_TICKET_SIZE_ERROR = "당첨 번호의 개수가 맞지 않습니다.";
     private static final String BONUS_DUPLICATE_ERROR = "당첨 번호에 보너스 번호가 포함되어 있습니다.";
     private final LottoTicket winningTicket;
     private final LottoNumber bonusNumber;
```

Comment:
```suggestion
    private static final String BONUS_DUPLICATE_ERROR = "당첨 번호에 보너스 번호가 포함되어 있습니다.";
    
    private final LottoTicket winningTicket;
    private final LottoNumber bonusNumber;
```
공백을 한칸 넣어주면 코드 읽기 더 수월할 거 같아요 :)